### PR TITLE
Tweaks to readme

### PR DESCRIPTION
**Pull request by mcabbott**
I think the `@time` examples in the readme were testing compilation time. And I wanted to upgrade the warning about replacing matrix exp to be a bit stronger. 

And while I was doing things I fixed up Project.toml a bit.

### CODE REVIEW

**Comment by Crown421** [comment]
Indeed, thank you.

**Comment by mcabbott** [comment]
I’m an idiot, when I wrote `julia = "0.7 1.0”` I should have put a comma. Without it you get warnings:
```
(v1.3) pkg> resolve
 Resolving package versions...
┌ Warning: julia version requirement for package VML not satisfied
```

File: Project.toml [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+version = "0.2.0"
 
 [deps]
 CpuId = "adafc99b-e345-5852-983c-f28acb93d879"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 
 [compat]
-julia = "≥ 0.7 1.0"
+julia = "0.7 1.0"
```
Comment (mcabbott) [inline Project.toml]:
```
julia = "0.7, 1.0"
```

**Comment by Crown421** [comment]
I added the comma manually.

**Pull request merged**